models/Checklist.js

```diff
@@ -8,14 +8,15 @@ var Types = keystone.Field.Types;
 
 var Checklist = new keystone.List('Checklist', {
     map: { name: 'title.EN' },
-    autokey: { path: 'slug', from: 'title.EN', unique: false },
+    autokey: { path: 'slug', from: 'title.EN', unique: true },
 });
 
 Checklist.add({
     title: {
         EN: { type: String, required: true },
         HI: { type: String }
     },
+    descriptor : { type: String },  
```

Comment:
what is this?

Reply:
the text field for the checklist that adrija requested.